test/upsert/incident-49/03-reset.td

```diff
@@ -0,0 +1,8 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
```

Comment:
Why do we have this empty file? Just because we need to give testdrive something when we run?

Reply:
yeah :(